src/Framework/Framework/Binding/HelperNamespace/DateTimeExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace DotVVM.Framework.Binding.HelperNamespace
+{
+    public static class DateTimeExtensions
+    {
+
+        /// <summary>
+        /// Converts the date (assuming it is in UTC) to browser's local time.
+        /// CAUTION: When evaluated on the server, no conversion is made as we don't know the browser timezone.
+        /// </summary>
+        public static DateTime ToBrowserLocalTime(this DateTime value) => value;
```

Comment:
This is not nice! If you have BusinessPack export to excel etc... this will export the wrong data.

Reply:
What is your suggestion then?

If someone wants to generate an Excel file on the server, they will need to know the user's timezone. There is no standard way how to send it to the server, and we decided recently that we don't want to introduce such mechanism in the framework.

We can leave this to the BP users - maybe they will figure the timezone out somehow, or maybe exporting the dates in UTC makes sense in their case.

I just don't see why we should block this framework feature because of this. 
We cannot throw the exception here because many controls evaluate getters, and we want the server rendering to render the UTC date.

Reply:
After an internal discussion with @tomasherceg,  we could not find any suitable solution. So, it can go as it is.